### ISSUE TITLE
Accessibility audit: Sub navigation section hidden text heading missing

### ISSUE THREAD

**Issue by jonhurrell** [dwp/design-system#305]
## Describe the issue
We don't have a hidden heading in the sub navigation on pages to explain what the links a for e.g.: Pages in this section

## Fix Recommendation
- [ ] Add <h2 class="govuk-visually-hidden" id="app-subnav-heading">Pages in this section</h2> to sub navigation.
